types/videojs-hotkeys/index.d.ts

```diff
@@ -0,0 +1,46 @@
+// Type definitions for videojs-hotkeys 0.2
+// Project: https://github.com/ctd1500/videojs-hotkeys
+// Definitions by: James Cote <https://github.com/Coteh>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+import { VideoJsPlayer } from 'video.js';
+
+declare module 'video.js' {
+    interface VideoJsPlayer {
+        hotkeys(options: VideoJsHotkeysOptions): void;
```

Comment:
```suggestion
        hotkeys(options?: VideoJsHotkeysOptions): void;
```
this look like optional param:
https://github.com/ctd1500/videojs-hotkeys/blob/master/videojs.hotkeys.js#L64

Reply:
Think you're right. Default options are used if you don't pass any options. Thanks for pointing this out. :)